articles/what-is-a-custom-domain.mdx

```diff
@@ -0,0 +1,64 @@
+---
+_schema: default
+_uuid: 5804dd03-210e-48cc-a02f-1b29f439ce0e
+_created_at: 2024-11-22T14:15:34+13:00
+title: What is a Custom Domain?
+nav_title: What is a Custom Domain?
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: Learn about Custom Domains, subdomains, and subpaths.
+article_category:
+  - Explanation
+article_topic:
+  - Custom Domains
+tags: []
+related_articles: []
+related_links: []
+explicit_canonical:
+---
+A Custom Domain is a unique address used to identify a website. For example, CloudCannon's domain is [cloudcannon.com](https://cloudcannon.com/). The domain is the part of your URL that users are most likely to remember. You can purchase a Domain Name from a Domain Registrar.
```

Comment:
We use `purchase` a Domain Name and `rent` a domain name in different parts of the docs.
You technically always `renting` as you have to pay for it annually, but most places refer to it as `purchasing` a domain name. I think we should go with `purchase`.